src/main/java/pl/tomaszdziurko/jvm_bloggers/blogs/domain/Blog.java

```diff
@@ -63,4 +66,7 @@ public Boolean getDefaultApprovedValue() {
         }
     }
 
+    public String getStatus() {
+        return active ? "Active" : "Deactivated";
```

Comment:
I am not fan of getting status this way, so this is a just my opinion but isn't better to provide `enum` here? I know it is not used anywhere else except of printing status now, but if someone will want to compare this status has to check the what string values are and compare it.